.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/devcontainers/base:bookworm
+
+# taken from: https://github.com/LMS-Community/slimserver-platforms/blob/public/9.1/Docker/Dockerfile
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL="C.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8"
```

Comment:
Is there any way to configure this _easily_ (otherwise I'd just leave it alone).